### DISCUSSION TITLE
Number of history entries?

### DISCUSSION THREAD

**Discussion by ZPMU** [sumatrapdfreader/sumatrapdf#2780] [Q&A]
Hello alltogether,

is there an option to set (increase) the **number of remembered opened files**?
I'd like to set the history entries limit for example to 50 (provided that there is a sensible way of displaying all of them in the File-menu).

Regards, Peter

**Comment by GitHubRulesOK** [comment]
visible history is tied to key 1-0 thus 10 entries as historically used by home screen and other menu shortcuts, Favorites can be a longer listing and I used those in the past, there was an experimental use of vbkm (now dropped, that provided for nesting groups) 

currently the Ctrl+K list may offer more than 10 at a time

It is fairly easy to write a list in the operating system for your own use that calls SumatraPDF to open any of thousands of grouped pdfs but that's same as any document manager library usage.

**Reply by ZPMU** [reply]
Sorry, but what is Ctrl+K? There seems to be no obvious effect.
V3.2/64, german GUI.
Ciao, Peter

**Reply by ZPMU** [reply]
Sorry again, I'll try 3.4.6

**Reply by GitHubRulesOK** [reply]
Sorry I should have mentioned Ctrl+K is probably only in 3.4.x
